Documentation/Sources/templates/carousel.md

```diff
@@ -1,33 +1,44 @@
 | Notification Category | ✅ | `aps.category` | string | The notification’s type. This string must correspond to the identifier of one of the `UNNotificationCategory` objects you register at launch time.<br /><br />Value will always be "AEPNotification" to use an Adobe push template. |
 | Mutable content | ✅ | `aps.mutable-content` | number | The notification service app extension flag. If the value is 1, the system passes the notification to your notification service app extension before delivery. Use your extension to modify the notification’s content.<br /><br />Value must always equal 1 so that the app's notification content service is called prior to the notification being displayed to the user. |
 | Sound | ⛔️ | `aps.sound` | string | The name of a sound file in your app’s main bundle or in the Library/Sounds folder of your app’s container directory. Specify the string “default” to play the system sound. Use this key for regular notifications. For critical alerts, use the sound dictionary instead. |
 | Badge Count | ⛔️ | `aps.badge` | string | The number to display in a badge on your app’s icon. Specify 0 to remove the current badge, if any. |
 | Notification Thread ID | ⛔️ | `aps.thread-id` | string | An app-specific identifier for grouping related notifications. This value corresponds to the `threadIdentifier` property in the `UNNotificationContent` object. |
-| Content available | ⛔️ | `aps.content-available` | number | The background notification flag. To perform a silent background update, specify the value 1 and don’t include the alert, badge, or sound keys in your payload. |
-| Image | ⛔️ | `adb_media` | string | URI of an image to be shown when notification is expanded. |
-| Link URI | ⛔️ | `adb_uri` | string | URI to be handled when user clicks the notification body. <br /><br />For example, a deep link to your app or a URI to a webpage. |
-| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br /><br />Represented as six character hex, e.g. `00FF00` |
+
+### AEPNotificationContent properties
+
+| **Field** | **Required** | **Key** | **Type** | **Description** |
+| :-------- | :----------- | :------ | :------- | :-------------- |
+| Payload Version | ✅ | `adb_version` | string | Version of the payload assigned by the Adobe authoring UI. |
+| Template Type | ✅ | `adb_template_type` | string | Informs the reader which properties may exist in the template object.<br /><br />Carousel template uses a value of "car". |
+| Expanded Title | ⛔️ | `adb_title_ex` | string | Title of the message when the notification is expanded.<br /><br />If an expanded title is not provided, the value in `aps.alert.title` will be used. |
+| Expanded Body | ⛔️ | `adb_body_ex` | string | Body of the message when the notification is expanded.<br /><br />If an expanded body is not provided, the value in `aps.alert.body` will be used. |
```

Comment:
we can remove `Expanded Body` for Carousel. 
we use Carousel Items. txt as body for each carousel item

Reply:
right, thank you.